docs/developinstallwin.rst

```diff
@@ -0,0 +1,67 @@
+Advanced Installation (Windows)
+------------------------------
+
+This is a quick guide on how to install AvaFrame
+and the required dependencies on your machine. AvaFrame is developed on **Linux
+machines** (Ubuntu/Manjaro/Arch) with recent Python versions > 3.8.
+These instructions assume you are familiar with working in a terminal. This
+guide is described for **Windows**. For *Linux*, see :ref:`developinstall:Advanced Installation (Linux)`.
+
+Requirements
+^^^^^^^^^^^^
+
+Install `git <https://github.com/git-guides/install-git>`_ and python, we
+suggest to work with miniconda/anaconda. For installation see `miniconda
+<https://docs.conda.io/en/latest/miniconda.html>`_ or
+`anaconda <https://docs.anaconda.com/anaconda/install/linux/>`_.
+
+Install `Microsoft C++ compiler <https://wiki.python.org/moin/WindowsCompilers>`_.
+Follow the installation steps for the version corresponding to the installed python version.
+
+Setup AvaFrame
+^^^^^^^^^^^^^^
+
+Open conda shell and create a new `conda environment
+<https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html>`_
+for AvaFrame, activate it and install pip, numpy and cython in this environment::
+
+  conda create --name avaframe_env
+  conda activate avaframe_env
+  conda install pip numpy cython
+
+Clone the AvaFrame repository (in a directory of your choice: [YOURDIR]) and change into it::
+
+  cd [YOURDIR]
+  git clone https://github.com/avaframe/AvaFrame.git
+  cd AvaFrame
+
+Compile the cython com1DFA part::
+
+   python setup.py build_ext --inplace
+
+.. Warning::
+   You will have to do this compilation every time something changes in the cython code. We also suggest
+   to do this everytime updates from the repositories are pulled.
+
+   **Note**: After (re)compilation in Windows, delete AvaFrame/build directory, in addition to any .pyd, .c, and .pycache files in AvaFrame/avaframe/com1DFA
+
+Install avaframe and its requirements by **either** doing::
+
+  pip install -e .
+
+or if this fails (see `github issue 986 <https://github.com/avaframe/AvaFrame/issues/986>`_), do::
+
+  python setup.py develop
+
+This installs avaframe in editable mode, so every time you import avaframe the
+current (local) version will be used.
+
+Test it by starting ``python`` and do an ``import avaframe``. If no error comes
+up, you are good to go.
+
+If wanted: Integrate conda environment into Editor of your choice (i.e. PyCharm, Spyder or any other)
+
+Head over to :ref:`gettingstarted:First run` for the next steps.
+
+NOTE:
+  Installing AvaFrame on Windows via QGIS plugin writes AvaFrame to system-wide python. This will override any avaframe installation in conda environment!
```

Comment:
Mh, this is not entirely true but depends on the mode of installation. There is no such thing as system-wide python on Windows, only python installations that might be accessed/shared. I suggest rewriting it to something like:
```suggestion
  Installing AvaFrame on Windows via QGIS plugin might overrule your script AvaFrame installation! This depends on your mode of installation (conda, OSGeo4W, direct installer...).
```